recipes/locust/run_test.py

```diff
@@ -0,0 +1 @@
+from locust import HttpLocust, TaskSet, task
```

Comment:
This could be brought into the `meta.yaml` as:

```
test:
   imports:
      - locust
      - locust.rpc
      - locust.util
```

Just importing `locust` gets the three submodules automatically.